web/main/browser.ts

```diff
@@ -11,15 +11,28 @@ import {
 
 import { locale } from '../stores/locale.js'
 
+const actualizeThemeColor = (themeColorTag: Element | null): void => {
+  let backgroundColor = window
+    .getComputedStyle(document.body)
+    .getPropertyValue('background-color')
+
+  if (themeColorTag && backgroundColor) {
+    themeColorTag.setAttribute('content', backgroundColor)
```

Comment:
Using var before it initialization is not good for readability.

Just move `actualizeThemeColor` below `let themeColorTag`

Reply:
Moved it below the let declaration, renamed the function param to `themeNode` — eslint complained on having the same name as the already declared variable above

Reply:
Actually, there's one more option — we can make the function not accepting params and just working with the `themeTag` from the scope, does it sound good?